### ISSUE TITLE
Right project for Windows UWP / makeappx features?

### ISSUE THREAD

**Issue by tuxuser** [rust-mobile/xbuild#175]
Hey,

I was wondering if this is the most advanced/recommended tool when it comes to packaging for Windows (UWP)? The uncertainty on my side comes from the organization-name `rust-mobile`.

Other projects however, like

cargo-msix
cargo-uwp

are not as advanced as cargo-xbuild.

The reason of the question being that I could contribute the following features:

- Platform-independent En/Decryption support for emsix (eappx)
- Publisher Id generation

And maybe others, like (e)msixbundle/eappxbundle support - depending on how fast I can progress with the other mentioned tasks.

In that regard, a standalone CLI invocation for the following features would be well-suited imo, like: unpack, unbundle, decrypt, pack, bundle, encrypt.

TL;DR Is this the correct project for the mentioned features?

**Comment by dvc94ch** [comment]
Also does windows packaging. The main differentiator is we try to do cross packaging (can build a windows msix package on Linux or macos). This means supporting new features is more work (can't just assume the host machine is Windows with all the msix cli tools pre installed)

**Comment by tuxuser** [comment]
Perfect, that's exactly my target/goal too.

**Comment by dvc94ch** [comment]
Whatever you think works best. I don't really have much time anyway
